{
  super.handleIQResult(iq);
  try {
    handleIQ(iq);
  }
 catch (  Exception e) {
    logd("An error occurred while trying to handle result IQ.");
    loge(e);
  }
}
